packages/nodes-base/credentials/OktaApi.credentials.ts

```diff
@@ -0,0 +1,47 @@
+import type {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class OktaApi implements ICredentialType {
+	name = 'oktaApi';
+
+	displayName = 'Okta API';
+
+	icon = 'file:Okta.svg';
+
+	properties: INodeProperties[] = [
+		{
+			displayName: 'Okta Domain',
+			name: 'oktaDomain',
```

Comment:
This should probably not contain "okta" and just be "domain" as it is already in the Okta credentials, and so is implied. Also is it actually more an URL than a domain as it starts with https.

Reply:
Yep some of the credentials wording are not consistent with what we have already because there were generated programmatically. I need to do a bit of some manual work to polish some of them.

Reply:
Ah nice, did not know that. Makes sense. :+1: